tiflash/tiflash-configuration.md

```diff
@@ -212,6 +212,14 @@ delta_index_cache_size = 0
     ## New in v6.2.0. This item specifies the minimum ratio of valid data in a PageStorage data file. When the ratio of valid data in a PageStorage data file is less than the value of this configuration, GC is triggered to compact data in the file. The default value is 0.5.
     dt_page_gc_threshold = 0.5
 
+    ## New in v7.0.0. This item specifies the maximum memory available for the HashAggregation operator with group by key before a spill is triggered. When the memory usage exceeds the threshold, HashAggregation reduces memory usage by spilling to disk. This item defaults to 0, which means that the memory usage is unlimited.
+    max_bytes_before_external_group_by = 0
+
+    ## New in v7.0.0. This item specifies the maximum memory available for the sort or topN operator before a spill is triggered. When the memory usage exceeds the threshold, the sort or topN operator reduces memory usage by spilling to disk. This item defaults to 0, which means that the memory usage is unlimited.
```

Comment:
```suggestion
    ## New in v7.0.0. This item specifies the maximum memory available for the sort or topN operator before a disk spill is triggered. When the memory usage exceeds the threshold, the sort or topN operator reduces memory usage by spilling to disk. This item defaults to 0, which means that the memory usage is unlimited and spill to disk is never used for topN.
```

Reply:
Should be "spill to disk is never used for sort or TopN"